tests/e2e/gauntlet_deployer.go

```diff
@@ -0,0 +1,370 @@
+	networkConfig := map[string]string{
+		"NODE_URL":          nodeUrl.String(),
+		"CHAIN_ID":          "localterra",
+		"DEFAULT_GAS_PRICE": "1",
+		"MNEMONIC":          "symbol force gallery make bulk round subway violin worry mixture penalty kingdom boring survey tool fringe patrol sausage hard admit remember broken alien absorb",
+	}
+
+	return networkConfig
+}
+
+// UpdateReportName updates the report name to be used by gauntlet on completion
+func UpdateReportName(reportName string, g *gauntlet.Gauntlet) {
+	g.NetworkConfig["REPORT_NAME"] = filepath.Join(utils.Reports, reportName)
+	err := g.WriteNetworkConfigMap(utils.Networks)
+	Expect(err).ShouldNot(HaveOccurred(), "Failed to write the updated .env file")
+}
+
+// GetInspectionResultsFromOutput parses the inpsectiond data from the output
+//  TODO we should really update the inspection command to just output json in the future
+func GetInspectionResultsFromOutput(output string) (map[string]InspectionResult, error) {
+	lines := strings.Split(output, "\n")
+	passRegex, err := regexp.Compile("✅  (.+) matches: (.+)$")
+	if err != nil {
+		return map[string]InspectionResult{}, err
+	}
+	failRegex, err := regexp.Compile("⚠️   (.+) invalid: expected (.+) but actually (.*)$")
+	if err != nil {
+		return map[string]InspectionResult{}, err
+	}
+	results := map[string]InspectionResult{}
+	for _, l := range lines {
+		passMatches := passRegex.FindStringSubmatch(l)
+		failMatches := failRegex.FindStringSubmatch(l)
+		if len(passMatches) == 3 {
+			results[passMatches[1]] = InspectionResult{
+				Pass:     true,
+				Key:      passMatches[1],
+				Expected: "",
+				Actual:   passMatches[2],
+			}
+		} else if len(failMatches) == 4 {
+			results[failMatches[1]] = InspectionResult{
+				Pass:     false,
+				Key:      failMatches[1],
+				Expected: failMatches[2],
+				Actual:   failMatches[3],
+			}
+		}
+	}
+
+	return results, nil
+}
+
+// LoadReportJson loads a gauntlet report into a generic map
+func LoadReportJson(file string) (map[string]interface{}, error) {
+	jsonFile, err := os.Open(filepath.Join(utils.Reports, file))
+	if err != nil {
+		return map[string]interface{}{}, err
+	}
+	defer jsonFile.Close()
+
+	byteValue, err := ioutil.ReadAll(jsonFile)
+	if err != nil {
+		return map[string]interface{}{}, err
+	}
+
+	var data map[string]interface{}
+	err = json.Unmarshal([]byte(byteValue), &data)
+
+	return data, err
+}
+
+// GetTxAddressFromReport gets the address from the typical place in the json report data
+func GetTxAddressFromReport(report map[string]interface{}) string {
+	return report["responses"].([]interface{})[0].(map[string]interface{})["tx"].(map[string]interface{})["address"].(string)
+}
+
+// DeployToken attempts to deploy the link token, currently does not work very often and I
+//  have never found a good reason why - Tate
+func (gd *GauntletDeployer) DeployToken() {
+	// TODO figure out why this never passes??? for a future pr
```

Comment:
What's the exact issue here?

Reply:
I started triaging into that again last night. I think I have figured it out and will push the fix with this pr. The deploy function is taking a codeIds argument but never using it in favor of just using hard coded ids for the cw20_base. 

I will add the ability to pull from a codeids file if the hardcoded network does not exist and add a prompt stating so when it does occur.